hazelcast/src/test/java/com/hazelcast/spi/impl/operationexecutor/slowoperationdetector/SlowOperationDetectorAbstractTest.java

```diff
@@ -113,10 +101,6 @@ static void shutdownOperationService(HazelcastInstance instance) {
         return invocationMap.values();
     }
 
-    static InternalOperationService getInternalOperationService(HazelcastInstance instance) {
-        return TestUtil.getNode(instance).nodeEngine.getOperationService();
-    }
-
     static int getDefaultPartitionId(HazelcastInstance instance) {
```

Comment:
Please have a look in IDEA, that method is already in master. It may be confusing because it is not 100% easy what I have to do there. I totally don't care about keys and values in my tests. I just need the partitionId of the one entry which is in the map for a nested operation with different partitionIds.